# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gtkextra/fix_configure_in.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./configure.in.old	Mon Jul 22 19:09:14 2002
+++ ./configure.in	Sun Feb  2 22:46:21 2003
@@ -4,7 +4,7 @@
 # Save this value here, since automake will set cflags later
 cflags_set=${CFLAGS+set}
 
-AC_DIVERT_PUSH(AC_DIVERSION_NOTICE)dnl
+AC_DIVERT_PUSH(NOTICE)dnl
 # Making releases:
 #   GTK_EXTRA_MICRO_VERSION += 1;
 #   GTK_EXTRA_INTERFACE_AGE += 1;
@@ -20,7 +20,7 @@
 GTK_EXTRA_BINARY_AGE=0
 GTK_EXTRA_VERSION=$GTK_EXTRA_MAJOR_VERSION.$GTK_EXTRA_MINOR_VERSION.$GTK_EXTRA_MICRO_VERSION
 dnl
-AC_DIVERT_POP()dnl
+AC_DIVERT_POP()dnl
 
 AC_SUBST(GTK_EXTRA_MAJOR_VERSION)
 AC_SUBST(GTK_EXTRA_MINOR_VERSION)
